src/cmds/fs/ls.c

```diff
@@ -168,27 +168,40 @@ int main(int argc, char **argv) {
 	if (optind < argc) {
 		struct stat sb;
 
-		if (-1 == stat(argv[optind], &sb)) {
-			return -errno;
-		}
+		optind--;
 
-		if (!S_ISDIR(sb.st_mode)) {
-			printer(argv[optind], &sb);
-			return 0;
-		}
+		while (optind++ < argc - 1) {
+
+		    if (-1 == stat(argv[optind], &sb)) {
+			    return -errno;
+            }
+
+            if (!S_ISDIR(sb.st_mode)) {
+                printer(argv[optind], &sb);
+                continue;
+		    }
```

Comment:
Can I please ask you to follow our code style - https://github.com/embox/embox/wiki/Code-Style. In this part of code something wrong with tabs/spaces.

Reply:
The same below.

Reply:
yeah，I also found it but it looks right on my computer. Is it because of a mix of tabs and spaces?

Reply:
yes, most likely because of that